README.md

```diff
@@ -47,6 +59,8 @@ You can also provide an absolute path:
 eslint-find-new-rules ~/Developer/eslint-config-kentcdodds/index.js
 ```
 
+**Please note** that any tested ESLint config file must reside below your project's root.
```

Comment:
Ah, that makes sense. Thanks for explaining it. Maybe you could put this explanation at the bottom of the README or in another `NOTES.md` file or something. I don't expect many people to have this problem, but if they do it'd be good to be able to reference it easily.